broadcast/src/linked/parsed.rs

```diff
@@ -25,8 +25,12 @@ pub enum Error {
     InvalidPartial,
     #[error("Invalid threshold")]
     InvalidThreshold,
-    #[error("Cryptographic error: {0}")]
-    Crypto(#[from] commonware_cryptography::Error),
+    #[error("Sequencer parse error: {0}")]
```

Comment:
I'm ok to break this apart but think we should keep the inner error type rather than casting to a string.